experimental/dds/tree2/src/feature-libraries/sequence-field/format.ts

```diff
@@ -223,9 +223,22 @@ export interface HasRevisionTag {
 }
 export const HasRevisionTag = Type.Object({ revision: Type.Optional(RevisionTagSchema) });
 
+export interface Transient {
+	/**
+	 * When populated, the content is inserted/revived then deleted.
```

Comment:
It seems slightly confusing to say "when populated," as the field is not optional on this interface.